text/0084-general-transaction-extrinsic-format.md

```diff
@@ -0,0 +1,72 @@
+# RFC-0084: General transactions in extrinsic format
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 12 March 2024                                                                               |
+| **Description** | Support more extrinsic types by updating the extrinsic format                               |
+| **Authors**     | George Pisaltu                                                                              |
+
+## Summary
+
+This RFC proposes the expansion of the extrinsic type identifier from the current most significant bit of the first byte of the encoded extrinsic to the two most significant bits of the first byte of the encoded extrinsic.
+
+## Motivation
+
+In order to support transactions that use different authorization schemes than currently exist, a new extrinsic type must be introduced alongside the current signed and unsigned types. Currently, an encoded extrinsic's first byte indicate the type of extrinsic using the most significant bit - `0` for unsigned, `1` for signed - and the 7 following bits indicate the extrinsic format version, which has been equal to `4` for a long time.
+
+By taking one bit from the extrinsic format version encoding, we can support 2 additional extrinsic types while also having a minimal impact on our capability to extend and change the extrinsic format in the future.
```

Comment:
Going a bit more in the details here. There is a reference below, but what would be good to have right at the RFC (in my opinion):

1. What is the concrete usecase/what do we want to achieve and why? What benefits does it bring? "Different authorization" schemes is pretty generic ... what is wrong with the existing ones?
2. What other ways of adding these different authorization schemes have been considered? E.g. Why are they not covered by "unsigned"?

Reply:
I tried to answer your questions in the new iteration of `Motivation`. Let me know what you think.

Reply:
Thank you! Better proxy accounts - got it ;-) Now that's a good motivation!